sources/us/in/st_joseph.json

```diff
@@ -32,6 +32,7 @@
             "pattern": "^(?:[0-9]+ )(.*)",
             "replace": "$1"
         },
-        "type": "shapefile-polygon"
+        "type": "shapefile-polygon",
+        "postcode": "prop_zip"
```

Comment:
This source appears to be hidden behind a login so it is currently broken and will remain so until someone finds an alternate way to get the data.